webapp/package.json

```diff
@@ -26,6 +26,7 @@
     "@babel/preset-env": "7.21.5",
     "@babel/preset-react": "7.18.6",
     "@babel/preset-typescript": "7.21.5",
+    "@calls/common": "github:mattermost/calls-common#v0.26.0",
```

Comment:
Could you move this to the dependencies in `channels/package.json` instead? It's only used by the web app itself, so it should live there

Reply:
Also, have you thought about publishing this package to NPM? Git dependencies like to cause problems sometimes, and in this case, it'll cause NPM to install the `devDependencies` of `calls-common` which could conflict with the ones that the web app uses

Reply:
> Could you move this to the dependencies in channels/package.json instead? It's only used by the web app itself, so it should live there

Yeah, that was my fault, I didn't realize we had multiple ones until I had to generate the new translation strings. Then I forgot to port this.

> Also, have you thought about publishing this package to NPM? Git dependencies like to cause problems sometimes, and in this case, it'll cause NPM to install the devDependencies of calls-common which could conflict with the ones that the web app uses

I think we did at some point, @cpoile maybe you have more context?

Reply:
Ok, all done -- we can use it as `@mattermost/calls-common` now. 👍